### PR TITLE
Add: new program

### PR DESCRIPTION
**Pull request by sarthakbal0987**
I have added a java program to check the given number is an armstrong or not

### CODE REVIEW

File: src/PrepInsta Top 100 Codes/Getting Started/Check_Armstrong_Number.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        //Create a variable temp1 and initialize it to number.This variable is required to find the sum of number's digits raised to the power
+        // of the number of digits.
+        int temp1 = number;
+        //Create a variable power in which we will store the total of digits
+        int power = 0;
+        //Finding the total digits of the number
+        while (temp > 0) {
+            //create a variable remainder and initialize it to temp % 10
+            // remainder is nothing but a digit of the number
+            int remainder = temp % 10;
```
Comment (Asish-dot-Rout) [inline src/PrepInsta Top 100 Codes/Getting Started/Check_Armstrong_Number.java]: Here remainder is never used so write the while loop in such a way that remainder will not require.

**Review by Asish-dot-Rout — CHANGES_REQUESTED** [review]
Everything is fine 👍 just look at the one suggestion which i gave to optimize the code.

File: src/PrepInsta Top 100 Codes/Getting Started/Check_Armstrong_Number.java (hunk 1 above)
Comment (sarthakbal0987) [reply src/PrepInsta Top 100 Codes/Getting Started/Check_Armstrong_Number.java]: I have optimized the code according to your request

Comment (Dev-Soumyaranjan) [reply src/PrepInsta Top 100 Codes/Getting Started/Check_Armstrong_Number.java]: Thanks for optimization😊

**Review by Dev-Soumyaranjan — APPROVED** [review]
Now your code is  good to go for merging🎉

**Review by Asish-dot-Rout — APPROVED** [review]

**Pull request merged**